### ISSUE TITLE
Not getting correspondence games notification, if correspondence games are 160+

### ISSUE THREAD

**Issue by abunickabhi** [WandererXII/lishogi#172]
I lost the games [https://lishogi.org/TuuAfeENFYgO](url) and [https://lishogi.org/YK7chsBQgFJ9](url) with lishogi notifying me that the game is in progress and I need to make a move. I do get notifications for the rest of the 160+ correspondence games.

To pinpoint the bug, I go to person's profile, see the games he has played with me, and check each board to see, if the move has been made or not.

I have had this bug on lichess too, when the number of correspondence games were high. On lichess, it happened with 2-3 opponents, that lichess did not give notification.

In case of lishogi, only one opponent, patientzer0 I did not get the notification. And the weird thing is that, I have currently 10+ games with that opponent, but I did not get notified only in 2 of those games.

**Comment by justremember** [comment]
See https://github.com/ornicar/lila/issues/5283#issuecomment-511160948

**Comment by justremember** [comment]
Seems like 200+ is the new limit. So this is a problem with lichess too. Best to report mutually-occuring bugs in ornicar/lila first.

**Comment by abunickabhi** [comment]
Yes true, I have experienced this problem on lichess as well, and cases of me missing games in lichess are more. Lishogi has done it only once, and I got tilted since that was a winning position.

**Comment by justremember** [comment]
Until we find out the cause of this problem a temporary solution would be to lower the correspondence game limit.
